controllers/cloud.redhat.com/providers/job/provider.go

```diff
@@ -0,0 +1,15 @@
+package job
+
+import (
+	p "github.com/RedHatInsights/clowder/controllers/cloud.redhat.com/providers"
+)
+
+var ProvName = "job"
+
+func GetJob(c *p.Provider) (p.ClowderProvider, error) {
+	return NewJobProvider(c)
+}
+
+func init() {
```

Comment:
Now I see why you did this - to ensure you could go first - you would need to ensure ALL providers are bumped properly.